examples/cloud_run_cloudevents/send_cloudevent.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
```

Comment:
@grant is there a plan on how this repo will be used for samples vs. Cloud Functions samples? Just wondering how we avoid fragmentation and duplication.

Reply:
GCF samples should live in the python docs and generic samples (for http/CloudEvent) with the FF should live here.

I don't see much in terms of fragmentation looking at the samples. There aren't GCF samples that look too similar to this.

---

We should dedupe GCF vs Run samples in general, but that's a separate task.